### PR TITLE
Added win_iis_webapppooldefaults and win_dnsServerAddress module

### PR DESCRIPTION
**Pull request by adbertram**
This is a start at supporting IIS application pool defaults similar to the DSC resource https://github.com/PowerShell/xWebAdministration/tree/dev/DSCResources/MSFT_xWebAppPoolDefaults and a module for modifying DNS server addresses on a NIC.

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks @adbertram for this PR. This PR requires revisions, either because it fails to build or by reviewer request. Please make the suggested revisions. When you are done, please comment with text 'ready_for_review' and we will put this PR back into review.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by adbertram** [comment]
ready_for_review

**Comment by adbertram** [comment]
ready_for_review

**Comment by gregdek** [comment]
Thanks @adbertram for this new module. When this module receives 'shipit' comments from two community members and any 'needs_revision' comments have been resolved, we will mark for inclusion.

[This message brought to you by your friendly Ansibull-bot.]

File: windows/win_dnsServerAddress.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# this is a windows documentation stub.  actual code lives in the .ps1
+# file of the same name
+
+DOCUMENTATION = '''
+---
+module: win_dnsServerAddress
```
Comment (trondhindenes) [inline windows/win_dnsServerAddress.py:26]: we should use lower-case module names only

File: windows/win_dnsServerAddress.ps1 [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+        if ($compare = Compare-Object @compParams)
+        {
+            Write-Verbose -Message 'The DNS server search order does not match.'
+            $result.changed = $true
+        }
+    }
+    
+    if ($result.Changed)
+    {
+        Set-DnsClientServerAddress -InterfaceAlias $InterfaceAlias -ServerAddresses $Address
```
Comment (trondhindenes) [inline windows/win_dnsServerAddress.ps1:95]: PowerShell / OS Version requirements - is Set-DnsClientServerAddress available on PS 3.0 / 2008R2? If not we need to call that out explicitly in the docs.

File: windows/win_dnsServerAddress.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+'''
+
+RETURN = """
+
+"""
+
+EXAMPLES = r'''
+# This sets the primary DNS server address to 4.4.4.4 and the secondary to 8.8.8.8 on the Ethernet network adapter.
+$ ansible -i hosts -m win_dnsServerAddress -a "address='4.4.4.4','8.8.8.8' interfaceAlias='Ethernet'" all
+# Playbook example
```
Comment (trondhindenes) [inline windows/win_dnsServerAddress.py:69]: lower-case here as well.

File: windows/win_iis_webapppooldefaults.py [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+DOCUMENTATION = '''
+---
+module: win_iis_webapppooldefaults
+version_added: "2.2"
+short_description: Sets IIS settings for the application pool defaults
+description:
+     - Sets IIS settings for the application pool defaults
+options:
+  idleTimeout:
+    description:
```
Comment (trondhindenes) [inline windows/win_iis_webapppooldefaults.py]: We try and use "pythonic" parameters. For example "idle_timeout" instead of "idleTimeout" and such.

**Comment by ArieHein** [comment]
There already is a win_iis_webapppool module in the extra module that does less but wouldn't it be better to improve it, instead of writing a new one ?

**Comment by adbertram** [comment]
@ArieHein That one does not apply changes to the app pool defaults; only exist app pools.

**Comment by gregdek** [comment]
Thanks @adbertram for this PR. This PR requires revisions, either because it fails to build or by reviewer request. Please make the suggested revisions. When you are done, please comment with text 'ready_for_review' and we will put this PR back into review.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by ArieHein** [comment]
seems it was the ======= signes at line 27 in win_iis_webapppool_defaults.py 
that caused the failure as it expected --- instead :)

**Pull request closed without merging**